tests/version_test.py

```diff
@@ -34,10 +34,8 @@ def test_version_is_semver():
 def test_input_file_version(tmp_path):
     oct_init["geoh5"] = str(assets_path() / "FlinFlon.geoh5")
     params = OctreeParams(**oct_init)
-    params.version = "0.0.0"
+    params.version = "10.0.0"
     params.write_input_file("test.ui.json", tmp_path)
 
-    with pytest.warns(
-        UserWarning, match="Input file version '0.0.0' does not match the."
-    ):
+    with pytest.warns(UserWarning, match="Input file version '10.0.0' is ahead"):
```

Comment:
**suggestion:** have two other tests checking there is no warning for:
- current app version in file
- older app version in file, with something like:
```py
app_version = semver.Version.parse(geoapps.__version__)
if (app_version.minor > 0):
    version_in_file = app_version.replace(minor=app_version.minor-1)
else:
    version_in_file = app_version.replace(major=app_version.major-1)
```